run/ingress/main.tf

```diff
@@ -25,34 +25,26 @@ resource "google_project_service" "cloudrun_api" {
 }
 
 # [START cloudrun_service_ingress]
-resource "google_cloud_run_service" "default" {
+resource "google_cloud_run_v2_service" "default" {
   provider = google-beta
   name     = "ingress-service"
   location = "us-central1"
 
+  # For valid annotation values and descriptions, see
```

Comment:
Nice comment 😍 👏

Reply:
This was already in the file! But I can see how this sort of comment is very useful and this style is probably what should be added for the traffic enums